ci/jobs/build_clickhouse.py

```diff
@@ -172,8 +179,9 @@ def main():
                 with_log=True,
             )
         )
-        Shell.check("sccache --show-stats", verbose=True)
-        Shell.check(f"ls -l {build_dir}/programs/", verbose=True)
+        run_shell("sccache stats", "sccache --show-stats")
+        run_shell("clang-tidy-cache stats", "clang-tidy-cache --show-stats")
```

Comment:
How about printing it only for tidy builds?

Reply:
Done in [`5dda8bc` (#78492)](https://github.com/ClickHouse/ClickHouse/pull/78492/commits/5dda8bc774433ee9051eac72c48df216811fefe6)